lib/pbench/server/database/alembic/versions/1a91bc68d6de_update_api_key.py

```diff
@@ -20,11 +20,14 @@ def upgrade() -> None:
     op.drop_constraint("api_keys_pkey", "api_keys", type_="primary")
     op.execute("ALTER TABLE api_keys ADD COLUMN id SERIAL PRIMARY KEY")
     op.add_column("api_keys", sa.Column("name", sa.String(length=128), nullable=True))
-    op.create_unique_constraint("api_keys_api_key_key", "api_keys", ["api_key"])
+    op.add_column("api_keys", sa.Column("key", sa.String(length=500), nullable=False))
+    op.create_unique_constraint(None, "api_keys", ["key"])
+    op.drop_column("api_keys", "api_key")
```

Comment:
I don't think that `alter_column` handles demoting the previous primary key and elevating a new one, which I believe is what led to dropping the one and creating the replacement.

If we were really worrying about upgrading live databases, though, this would get tricky because dropping the column would lose existing API keys; whereas I think we'd want to retain them but *add* row `id` values and some boilerplate `label` to each row.

At this point, however, there's no data to retain and we're fine.